src/os_dependent/FSNodePOSIX.cxx

```diff
@@ -230,5 +230,10 @@ bool AbstractFilesystemNode::dirExists(const string& path)
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 bool AbstractFilesystemNode::makeDir(const string& path)
 {
-  return mkdir(path.c_str(), 0777) == 0;
+  return mkdir(path.c_str()
```

Comment:
I believe `FSNodePOSIX.cxx` shouldn't be compiled under Windows, and looking at your change to CMakeLists.txt that should also be true for mingw. Can you verify that this edit is still necessary?